### DISCUSSION TITLE
Shader Settings import/export

### DISCUSSION THREAD

**Discussion by jtsage** [StjerneIdioten/I3D-Blender-Addon#147] [Ideas & Feature Requests]
I have no idea how it could be implemented, but it really kind of hurts to have to retype all of the colorMask settings when you accidentally change the variant of the shader on a material.  Or, for instance, are converting something over to 22.

**Comment by StjerneIdioten** [comment]
For the "accidentally changing"-part, are you aware that the shader picker is entirely integrated with Blender's undo history? So if you accidentally change it, just ctrl-z your way until it's back 😉 

For converting something and changing shader version (FS19 --> FS22), I would have to program some sort or converter and honestly I don't know if that is worth the hassle for now (I do hope to have some form of auto-conversion going on between FS versions)

**Comment by ssnd292** [comment]
My opinion on this is to not make it convertable to FS22.

Since this version is basically "legacy" and where more or less lucky that it simply works with FS22, I dont think Stjerne should support this conversion.

**Reply by StjerneIdioten** [reply]
No worries, suggestions in here are not only for the legacy version 😄  I will transfer relevant suggestions into the other repositories once it is mature enough for me to consider them 😉
